app/views/help/registration_closed.html.haml

```diff
@@ -0,0 +1,18 @@
+- content_for :page_title do
+  = AppConfig.settings.pod_name + " - " + t("devise.shared.links.sign_up_closed")
+
+.container#login
+  .text-center
+    .logos-asterisk
```

Comment:
It looks like this doesn't work, no asterisk is displayed. I would be nice to have a design similar to the registration page, with the dandelion on the left and the text on the right.